system/system_monitor/reader/traffic_reader/traffic_reader_service.cpp

```diff
@@ -120,6 +121,9 @@ void TrafficReaderService::handle_message(const char * buffer)
     case Request::GET_RESULT:
       get_result();
       break;
+    case Request::STOP_NETHOGS:
+      stop_nethogs();
```

Comment:
A Same point. `skip_nethogs`, `SKIP_NETHOGS`

Reply:
Fixed in fad266d995c46e12f8ff763e4db67cb19a64298d